nodes/solid/solid_section.py

```diff
@@ -0,0 +1,112 @@
+# This file is part of project Sverchok. It's copyrighted by the contributors
+# recorded in the version control history of the file, available from
+# its original location https://github.com/nortikin/sverchok/commit/master
+#
+# SPDX-License-Identifier: GPL3
+# License-Filename: LICENSE
+
+import bpy
+from bpy.props import BoolProperty, EnumProperty
+
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import (updateNode, map_recursive, zip_long_repeat,
+                                     ensure_nesting_level, get_data_nesting_level)
+from sverchok.utils.curve.freecad import SvSolidEdgeCurve
+from sverchok.dependencies import FreeCAD
+
+if FreeCAD is not None:
+    from sverchok.utils.solid import SvBoolResult
+    import Part
+
+class SvSolidSectionNode(SverchCustomTreeNode, bpy.types.Node):
+    """
+    Triggers: Section
+    Tooltip: Generate Intersection Curves and Points
+    """
+    bl_idname = 'SvSolidSectionNode'
+    bl_label = 'Solid Section'
+    bl_icon = 'OUTLINER_OB_EMPTY'
+    sv_icon = 'SV_SOLID_SECTION'
+    sv_category = "Solid Operators"
+    sv_dependencies = {'FreeCAD'}
+
+    nurbs_output : BoolProperty(
+        name = "NURBS Output",
+        description = "Output curves in NURBS representation",
+        default = False,
+        update=updateNode)
+
+    def draw_buttons_ext(self, context, layout):
+        self.draw_buttons(context, layout)
+        layout.prop(self, 'nurbs_output', toggle=True)
+
+    def sv_init(self, context):
+        self.inputs.new('SvSolidSocket', "Shape A")
+        self.inputs.new('SvSolidSocket', "Shape B")
+        
+        self.outputs.new('SvVerticesSocket', "Vertices")
+        self.outputs.new('SvCurveSocket', "Curves")
+
+    def make_solid(self, solids):

```

Comment:
do we really need this method? It seems like it's possible to combine it with `make_solid_simple` into one method.

Reply:
Also I'm not sure if wrapping the result into `SvBoolResult` makes sense here. In Solid Boolean that class is used to bear information about which edge / face in the resulting object come from which original object. But here, as far as I understand, such information is not available anyway.

Reply:
Thanks! Noted and updated.